### ISSUE TITLE
Add sample with VueJS integration

### ISSUE THREAD

**Issue by alexsandro-xpt** [dumptyd/treeflex#1]
Add suport to a dynamic add and remove node element at tree.

**Comment by dumptyd** [comment]
Are you saying you'd like to see a Vue wrapper on top of this?

**Comment by alexsandro-xpt** [comment]
@dumptyd yes, if possible with greensock animate at event click to show and hide childrens.
